__tests__/app.test.js

```diff
@@ -0,0 +1,50 @@
+const request = require('supertest');
+const db = require('../db/connection');
+const app = require('../app');
+const seed = require('../db/seeds/seed');
+const { articleData, commentData, topicData, userData} = require('../db/data/test-data/index');
```

Comment:
No real need to destructure this data out when you're passing the whole object to seed on line 11 , could just import this into one variable and pass that in